scripts/get-coreos

```diff
@@ -41,12 +54,12 @@ echo "coreos_production_pxe_image.cpio.gz.sig"
 curl -# $BASE_URL/coreos_production_pxe_image.cpio.gz.sig -o $DEST/coreos_production_pxe_image.cpio.gz.sig
 
 # Install image
-echo "coreos_production_image.bin.bz2"
-curl -# $BASE_URL/coreos_production_image.bin.bz2 -o $DEST/coreos_production_image.bin.bz2
-echo "coreos_production_image.bin.bz2.sig"
-curl -# $BASE_URL/coreos_production_image.bin.bz2.sig -o $DEST/coreos_production_image.bin.bz2.sig
+echo $IMAGE_NAME
```

Comment:
Let's always download the default image (even if an oem is specified). I can already see a Tectonic user using an oem option and then being confused when the Installer app doesn't work since it only supports the default image.